package-lock.json

```diff
@@ -10,7 +10,7 @@
 			"integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```

Comment:
`package-lock.json` shouldn't be updated by this issue.

Reply:
Please, how can I avoid this should happens again?
Can you solve this problem or have I to perform any step?

Thanks.

Reply:
You can use `npm ci` instead `npm install` to avoid modifications in package-lock.json

Reply:
Ok got it.
What about the current situation to close this pull request? What can I do?
Thanks.

Reply:
You can revert changes for this file or create new PR without it.